src/Components/ActivationKeysTable/ActivationKeysTable.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  TableComposable,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+} from '@patternfly/react-table';
+import useActivationKeys from '../../hooks/useActivationKeys';
+import Loading from '../LoadingState/Loading';
+import Unavailable from '@redhat-cloud-services/frontend-components/Unavailable';
+
+const ActivationKeysTable = () => {
+  const columnNames = {
+    name: 'Key Name',
+    role: 'Role',
+    serviceLevel: 'SLA',
+    usage: 'Usage',
+  };
+  const { isLoading, error, data } = useActivationKeys();
+
+  const Results = () => {
```

Comment:
I would try to avoid defining components in components, as it does degrade rendering performance a bit.
I'd rather define the component outside and pass in the `data` as a prop.